docs/management/advanced-options.asciidoc

```diff
@@ -38,6 +38,7 @@ working on big documents. Set this property to `false` to disable highlighting.
 `courier:maxSegmentCount`:: Kibana splits requests in the Discover app into segments to limit the size of requests sent to
 the Elasticsearch cluster. This setting constrains the length of the segment list. Long segment lists can significantly
 increase request processing time.
+`courier:ignoreFilterIfFieldNotInIndex`:: Useful when dashboards consist of visualizations from multiple index patterns, this setting will skip filters that apply to fields that don't exist in the visualization's index.
```

Comment:
I assume this is a boolean setting. If so, consider wording it like other boolean settings, something like:

> `courier:ignoreFilterIfFieldNotInIndex`:: Set this property to `true` to skip filters that apply to fields that don't exist in a visualization's index. Useful when dashboards consist of visualizations from multiple index patterns.